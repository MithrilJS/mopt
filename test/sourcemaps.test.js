"use strict";

var code = require("./lib/code");

describe("Source Maps", function() {
    it("should output an accurate source map", function() {
        expect(
            code(`m(".fooga")`, { sourceMaps : "inline" })
        )
        .toMatchSnapshot();
    });
});
