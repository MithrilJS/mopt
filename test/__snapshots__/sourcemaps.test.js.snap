// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Source Maps should output an accurate source map 1`] = `
"m.vnode(\\"div\\",undefined,{className:\\"fooga\\"},[],undefined,undefined);
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbInVua25vd24iXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBQUEiLCJmaWxlIjoidW5rbm93biIsInNvdXJjZXNDb250ZW50IjpbIm0oXCIuZm9vZ2FcIikiXX0="
`;
